schema/settings/model.fbs

```diff
@@ -31,6 +31,7 @@ table ModelRatios{
 // Settings for leg tweaks
 table LegTweaksSettings{
     correction_strength: float32 = null;
+    enabled: bool = null;
```

Comment:
What exactly is being enabled here? Floor clip? Skating correction?

Reply:
All of the features in legtweaks so floor-clip, skating correction, toe-snap, and foot-plant. Basically just all the enhancements that could make it harder to manually tune proportions

Reply:
you should break that boolean down into one for each feature, rather than grouping them all by a single one.
This is because later if we decide we want to control one or the other, we want the ability to do that in a non-breaking way.

But furthermore - don't we already have the ability to set those? How else is the GUI currently controlling those settings?

Reply:
Yes we do have all those toggles already but using those runs the risk of the saving the wrong value to the config, like Erien mentioned. This avoids that problem. With that context in mind should I still split it in to separate bools?

Reply:
What does saving values to a config have to do with solarxr? This PR really doesn't make any sense to me. Maybe I'm missing some context? Lets discuss in discord to get on the same page.

Reply:
https://discord.com/channels/817184208525983775/1006626234685460651/1086757175445176411